packages/rum-recorder/src/boot/recorder.ts

```diff
@@ -13,6 +13,10 @@ export function startRecording(
   session: RumSession,
   parentContexts: ParentContexts
 ) {
+  // This needs to be invoked before startSegmentCollection so lifeCycle events are subscribed
+  // first, and ViewEnd records gets added before the segment is flushed.
```

Comment:
We also rely on the fact that `startRum` is called before `startRecording` to not generate a `ViewEnd` record with the initial view, all that seems fishy to me 😕 
What about having a `VIEW_ENDED` lifecycle event?

Reply:
Ah right, it is probably better.